python/xorbits/_mars/services/cluster/file_logger.py

```diff
@@ -30,6 +30,7 @@ class FileLoggerActor(mo.Actor):
     """
 
     def __init__(self):
+        super().__init__()
```

Comment:
What is this for?

Reply:
Sub class should always call the constructor of their parent class.